engine/src/main/java/org/destinationsol/game/BeaconHandler.java

```diff
@@ -138,7 +138,7 @@ private void updateTarget(SolGame game) {
             return;
         }
         if (farTarget == null) {
-            throw new AssertionError("Far target does not exist!");
+            throw new AssertionError("Far target " + farTarget + " does not exist!");
```

Comment:
If `farTarget` is null, printing it is not going to work.

Reply:
All right